sweep/sweeper.go

```diff
@@ -632,22 +632,24 @@ func (s *UtxoSweeper) collector(blockEpochs <-chan *chainntnfs.BlockEpoch) {
 func (s *UtxoSweeper) sweepCluster(cluster inputCluster,
 	currentHeight int32) error {
 
-	// Examine pending inputs and try to construct
-	// lists of inputs.
-	inputLists, err := s.getInputLists(cluster, currentHeight)
-	if err != nil {
-		return fmt.Errorf("unable to examine pending inputs: %v", err)
-	}
-
-	// Sweep selected inputs.
-	for _, inputs := range inputLists {
-		err := s.sweep(inputs, cluster.sweepFeeRate, currentHeight)
+	return s.cfg.Wallet.WithCoinSelectLock(func() error {
```

Comment:
should add a comment why coin select lock is needed here

Reply:
added